app/views/users/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<!-- <form action="/users" method="post" accept-charset="UTF-8">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <br><br>
+  <label for="username" >Username:</label>
+  <input type="text" id="username" name="user[username]" ><br><br>
+  <label for="email">Email:</label>
+  <input type="text" id="email" name="user[email]"><br><br>
+  <label for="password">Password:</label>
+  <input type="password" id="password" name="user[password]"><br><br>
+  <input type="submit" value="Submit">
+</form> -->
+<!-- <%= form_tag("/users", method: "post") do %>
+  <br><br>
+  <%= label_tag(:username, "Username:") %>
+  <%= text_field_tag(:username) %><br><br>
+  <%= label_tag(:email, "Email:") %>
+  <%= text_field_tag(:email) %><br><br>
+  <%= label_tag(:password, "Password:") %>
+  <%= password_field_tag(:password) %><br><br>
+  <%= submit_tag "Submit" %>
+<% end %> -->
+<!-- <%= form_with model: @user, method: :post do |form| %>
+  <br><br>
+  <%= form.label :username, "Username:" %>
+  <%= form.text_field :username, :value => 'Please enter your username here'%><br><br>
+  <%= form.label :email, "Email:" %>
+  <%= form.text_field :email, :value => 'user@domain.com' %><br><br>
+  <%= form.label :password, "Password:" %>
+  <%= form.password_field :password %><br><br>
+  <%= form.submit "Submit" %>
+<% end %>
```

Comment:
The `form_with` should not be commented out. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_with.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms))